### PR TITLE
Add basic eslint-plugin structure

### PR DESCRIPTION
**Pull request by flavius-tirnacop-hs**
## Need

Basic code structure
## Deliverables
- [x] Readme.md (update actual)
- [x] index.js - imports all rules in lib/rules
- [x] package.json
- [x] .eslintrc
- [x] lib/ - where the source code is stored
  - [x] rules/
- [x] docs/ - contains documentation
  - [x] rules/
- [x] test/
  - [x] lib/
    - [x] rules/ - tests for rules added in root/lib/rules
  - [x] moca.opts - mocha config file

### CODE REVIEW

**Comment by flavius-tirnacop-hs** [comment]
@andrei-picus-hs please review code. If we need anything else added tell me.

File: package.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  "description": "ESLint rules applicable to the uberVU org",
+  "keywords": [
+    "eslint",
+    "eslintplugin",
+    "eslint-plugin"
+  ],
+  "author": "uberVU",
+  "main": "index.js",
+  "scripts": {
+    "lint": "eslint --ext .js --ext .jsx --format './node_modules/eslint-friendly-formatter' .",
```
Comment (andrei-picus-hs) [inline package.json]: No JSX files here :smile:

File: package.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    "test": "mocha"
+  },
+  "dependencies": {
+    "requireindex": "~1.1.0"
```
Comment (andrei-picus-hs) [inline package.json]: What's this for?

File: package.json [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+  },
+  "devDependencies": {
+    "eslint": "1.5.1",
+    "eslint-friendly-formatter": "^1.1.0",
+    "mocha": "^2.0.1",
+    "pre-git": "^0.6.2"
+  },
+  "engines": {
+    "node": ">=0.10.0"
```
Comment (andrei-picus-hs) [inline package.json]: No need for this. We're not gonna use any engine specific features.

**Comment by andrei-picus-hs** [comment]
@flavius-tirnacop-hs please add a blank rule and a blank test to make sure everything is working.

File: package.json (hunk 2 above)
Comment (flavius-tirnacop-hs) [inline package.json]: We don't need this anymore. It was used in the initial index.js generated by https://www.npmjs.com/package/generator-eslint but I changed the structure a bit.

**Comment by flavius-tirnacop-hs** [comment]
@andrei-picus-hs I've updated the skeleton with a blank rule.

**Comment by andrei-picus-hs** [comment]
Too many blank lines in the code, but LGTM :smile:

**Pull request merged**